src/main/java/uk/gov/hmcts/divorce/citizen/notification/NocCitizenToSolsNotifications.java

```diff
@@ -0,0 +1,108 @@
+package uk.gov.hmcts.divorce.citizen.notification;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.divorce.divorcecase.model.CaseData;
+import uk.gov.hmcts.divorce.notification.ApplicantNotification;
+import uk.gov.hmcts.divorce.notification.CommonContent;
+import uk.gov.hmcts.divorce.notification.NotificationService;
+
+import static org.apache.commons.lang3.StringUtils.isNotBlank;
+import static uk.gov.hmcts.divorce.notification.EmailTemplateName.NOC_CITIZEN_TO_SOL_EMAIL_CITIZEN;
+import static uk.gov.hmcts.divorce.notification.EmailTemplateName.NOC_TO_SOLS_EMAIL_OLD_SOL;
+import static uk.gov.hmcts.divorce.notification.EmailTemplateName.NOC_TO_SOLS_EMAIL_SOL;
+
+@RequiredArgsConstructor
+@Component
+@Slf4j
+public class NocCitizenToSolsNotifications implements ApplicantNotification {
+
+    private final NotificationService notificationService;
+
+    private final CommonContent commonContent;
+
+
+    @Override
+    public void sendToApplicant1(final CaseData caseData, final Long id) {
+        log.info("Sending NOC notification to app1 : {}", id);
+
+        notificationService.sendEmail(
+            caseData.getApplicant1().getEmail(),
+            NOC_CITIZEN_TO_SOL_EMAIL_CITIZEN,
+            commonContent.nocCitizenTemplateVars(id, caseData.getApplicant1()),
+            caseData.getApplicant1().getLanguagePreference(),
+            id
+        );
+    }
+
+    @Override
+    public void sendToApplicant2(final CaseData caseData, final Long id) {
+        if (isNotBlank(caseData.getApplicant2EmailAddress())
+            && !caseData.getApplication().isSolicitorServiceMethod()) {
```

Comment:
I think it's effectively the bit of code that checks if this is sole application as opposed to joint and if the agreement is that solicitor is doing the part of contacting the respondent. But now I'm thinking that this does need to be different for the noc because there's now the solicitor for app2/respondent.
I'll just check if the email is not null or empty instead - cos this notification is about their sols changing and they need to be told.